sklearn/model_selection/partition.py

```diff
@@ -0,0 +1,1129 @@
+"""
+The :mod:`sklearn.model_selection.partition` module includes classes and
```

Comment:
I'll go ahead and do so since it was previously accepted... Please let me know if I should keep it as `validation` though :)